test/lint/lint-spelling.ignore-words.txt

```diff
@@ -11,5 +11,4 @@ nin
 ser
 unparseable
 unser
-useable
```

Comment:
Not sure why you've dropped this and changed usages to `usable`, so you can revert that as well.